lib/pbench/common/results.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import urllib.parse
+
+from logging import Logger
+from pathlib import Path
+from pbench import PbenchConfig
+
+
+class CopyResultTb:
```

Comment:
Do we have unit testing for this class?  (I don't see any updates relative to its move.)

Reply:
I have updated it